rust/datafusion/src/sql/planner.rs

```diff
@@ -437,6 +437,7 @@ pub fn convert_data_type(sql: &SQLType) -> Result<DataType> {
         SQLType::Float(_) | SQLType::Real => Ok(DataType::Float64),
         SQLType::Double => Ok(DataType::Float64),
         SQLType::Char(_) | SQLType::Varchar(_) => Ok(DataType::Utf8),
+        SQLType::Custom(t) if t.to_lowercase() == "string" => Ok(DataType::Utf8),
```

Comment:
We probably don't need this just yet, as `cast(binary as varchar)` works without any changes.